release/test_release_gold.sh

```diff
@@ -19,7 +26,11 @@ function set_up {
 	if [ ! -d "${_PROJECTS_DIR}/liferay-portal-ee" ]
 	then
 		echo -e "The directory ${_PROJECTS_DIR}/liferay-portal-ee does not exist.\n"
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
 	fi
+
+	export release_info_date="$(grep release.info.date= "${_PROJECTS_DIR}/liferay-portal-ee/release.properties")"
```

Comment:
@kiwm, shell variables with global scope should be declared using capital letters, like `_PROJECTS_DIR`, for instance. Can you please apply this pattern to the variable `release_info_date`?

Thanks